duoStrength.js

```diff
@@ -513,10 +513,11 @@ function addStrengths(strengths)
 			strengthBarHolder.className = "strengthBarHolder";
 			strengthBarHolder.style = 
 			`
-				width: 120%;
+				width: 100%;
 				height: 1.4em;
 				position: relative;
-				left: -10%;
+				// left: -10%;
+				transform: translate (0, 10%);

```

Comment:
This doesn't seem to be working on my machine, it doesn't show up in the styles for any of the elements:
![](https://toransharma.com/i/2019-08-19_23-38-30.png)

When I manually add the style in chrome, I think it looks worse, best to just remove it.

```suggestion
```